src/components/Autosuggest/Autosuggest.js

```diff
@@ -374,6 +412,12 @@ Autosuggest.propTypes = {
     noSuggestionsPlaceholder: PropTypes.string.isRequired,
     /** to be shown as clear button title */
     clearTitle: PropTypes.string,
+    /** input field ref */
+    inputRef: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+    /** root wrapper ref */
+    rootRef: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+    /** suggestions list ref */
+    listRef: PropTypes.object, // eslint-disable-line react/forbid-prop-types
```

Comment:
Can we try to define `PropTypes` in this way: https://stackoverflow.com/questions/48007326/what-is-the-correct-proptype-for-a-ref-in-react

Reply:
I wouldn't waste much time on it, if we are going to move it to TS soon

Reply:
Yes, it makes sence